packages/inter-protocol/src/vaultFactory/prioritizedVaults.js

```diff
@@ -144,50 +105,35 @@ export const makePrioritizedVaults = (store, higherHighestCb = () => {}) => {
       !AmountMath.isEmpty(vault.getCollateralAmount()),
       'Tracked vaults must have collateral (be liquidatable)',
     );
-    trace('addVault', key, 'when first:', firstKey);
-    if (!firstKey || keyLT(key, firstKey)) {
-      firstKey = key;
-      higherHighestCb(currentDebtToCollateral(vault));
-    }
     return key;
   };
 
-  /**
-   * Invoke a function for vaults with debt to collateral at or above the ratio.
-   *
-   * Results are returned in order of priority, with highest debt to collateral first.
-   *
-   * Redundant tags until https://github.com/Microsoft/TypeScript/issues/23857
-   *
-   * @param {Ratio} ratio
-   * @yields {[string, Vault]>}
-   * @returns {IterableIterator<[string, Vault]>}
-   */
-  // Allow generator function
-  // eslint-disable-next-line func-names
-  // eslint-disable-next-line no-restricted-syntax
-  function* entriesPrioritizedGTE(ratio) {
-    // TODO use a Pattern to limit the query https://github.com/Agoric/agoric-sdk/issues/4550
-    for (const [key, vault] of vaults.entries()) {
-      const debtToCollateral = currentDebtToCollateral(vault);
-      if (ratioGTE(debtToCollateral, ratio)) {
-        yield [key, vault];
-      } else {
-        // stop once we are below the target ratio
-        break;
-      }
+  const removeVaultsBelow = crKey => {
+    // crKey represents a collateralizationRatio based on the locked price.
+    // We'll remove all vaults below that ratio, and return them.
+
+    /** @type {MapStore<string, Vault>} */
+    const vaultsToLiquidate = makeScalarMapStore();
```

Comment:
this component doesn't know about "liquidation".
```suggestion
    const vaultsRemoved = makeScalarMapStore();
```

Reply:
done